apps/src/templates/teacherDashboard/LoginTypePicker.jsx

```diff
@@ -195,16 +226,48 @@ class LoginTypePicker extends Component {
             <WordLoginCard onClick={this.onLoginTypeSelect} />
             <EmailLoginCard onClick={this.onLoginTypeSelect} />
           </CardContainer>
-          {!hasThirdParty && (
-            <p style={{...style.mediumText, ...style.learnHow}}>
-              {i18n.thirdPartyProviderUpsell() + ' '}
-              <a href="https://support.code.org/hc/en-us/articles/115001319312-Setting-up-sections-with-Google-Classroom-or-Clever">
-                {i18n.learnHow()}
-              </a>
-              {' ' + i18n.connectAccountThirdPartyProviders()}
-            </p>
-          )}
         </div>
+        {experiments.isEnabled(experiments.SECTION_CREATE_LMS_CARDS) &&
+          !withAllLmsProviders && (
+            <>
+              <h6 style={style.subheader}>{i18n.lmsIntegrations()}</h6>
+              <div
+                style={style.lmsInfoCardsContainer}
+                data-testid={'lms-info-cards-container'}
+              >
+                {!withClever && (
```

Comment:
nit/optional: we can probably make some sort of map data structure to better represent this. Something like:

```
const providerConfig = {
  canvas: {
    lmsName: LmsLoginTypeNames.clever,
    lmsLogo: cleverLogo,
   lmsInformationUrl: LmsLoginInstructionUrls.clever
  }
}
```

Then in this section we can loop through `provider` and render out as needed (might have some other complications with sorting) but this would change it to a data structure config to rendering pattern instead of having the rendering logic depend on the LMS specific implementation.

Reply:
Yeah that's a great suggestion - I went with this implementation to be consistent with the existing patterns in the file, but in retrospect that probably isn't the cleanest implementation.

Reply:
Gonna merge as-is and look into improving this later. The providers at the top work well with just rendering out the providers from a config object, but the new cards are actually the providers not present in the config, so it adds a decent amount of extra logic.